script-base.js

```diff
@@ -1135,3 +1136,10 @@ Generator.prototype.removefolder = function(folder) {
         shelljs.rm("-rf", folder);
     }
 }
+
+Generator.prototype.installedVersionIsLessThan = function(version, installedJhipsterVersion) {
```

Comment:
this method name is bit confusing can we have `isVersionLessThan` or something like that